tests/e2e/utils/PreferencesHandler.ts

```diff
@@ -0,0 +1,35 @@
+
+import { injectable, inject } from 'inversify';
+import { Logger } from './Logger';
+import { CLASSES } from '../inversify.types';
+import { RequestHandler } from './RequestHandler';
+import { TestConstants } from '../TestConstants';
+import { RequestType } from './RequestType';
+
+@injectable()
+export class PreferencesHandler {
+
+    constructor(@inject(CLASSES.RequestHandler) private readonly requestHandler: RequestHandler) {
+    }
+
+    public async setTerminalToDom() {
```

Comment:
Method implementation looks overflowed, please try to clean up method and make it more readable.
(Simplifying logic, extracting steps to the atomic private methods ...)

Reply:
yeah... I would actually do two changes here.
1) I would change this method to be more general (something like `setTerminalType(string/enum)` so we can switch back and forth if needed. This shouldn't be a big change.
2) I would also prefer separation to for example two methods - this one would be handling the requests (get/post) and second one would be handling the work with json (parsing&editing the json object)

Reply:
I've simplified the logic, so now I don't think we need to separate it to methods. WDYT? Can you please review it again?